packages/ffe-grid-react/src/index.d.ts

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+
+export interface GridProps extends React.HTMLAttributes<HTMLElement> {
+    children: React.ReactNode;
+    className?: string;
+    condensed?: boolean;
+    element?: string;
+    topPadding?: boolean;
+}
+
+export interface InlineGridProps extends React.HTMLAttributes<HTMLElement> {
+    children: React.ReactNode;
+    className?: string;
+    element?: string;
+}
+
+type backgroundColors =
```

Comment:
Nitpicking but I think it is normal to use PascalCase also for types.

Reply:
I'll fix :)